<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImuView</class>
 <widget class="QWidget" name="ImuView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBoxFileLogging"/>
   </item>
   <item row="0" column="5" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Network Logging</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBoxNetworkLogging">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinBoxNetworkLogging"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBoxNetworkLogging"/>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBoxFileLogging">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEditNetworkLogging"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File Logging</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Gravity Model</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBoxGravityModel"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxFileLogging</tabstop>
  <tabstop>comboBoxFileLogging</tabstop>
  <tabstop>checkBoxNetworkLogging</tabstop>
  <tabstop>comboBoxNetworkLogging</tabstop>
  <tabstop>lineEditNetworkLogging</tabstop>
  <tabstop>spinBoxNetworkLogging</tabstop>
  <tabstop>comboBoxGravityModel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
